.env

```diff
@@ -0,0 +1,6 @@
+POSTGRES_USER=leechcoder
```

Comment:
instead of pushing the .env, could you push a .env.sample
for server we can keep the naming .env